charts/cluster/templates/_helpers.tpl

```diff
@@ -68,3 +68,18 @@ If a custom imageName is available, use it, otherwise use the defaults based on
         {{ fail "Invalid cluster type!" }}
     {{- end }}
 {{- end -}}
+
+{{/*
+Cloud backups enabled
+*/}}
+{{- define "cluster.cloudBackupsEnabled" -}}
+{{- $cloudProviders := list "s3" "azure" "google" -}}
+{{- and .Values.backups.provider (has .Values.backups.provider $cloudProviders) -}}
+{{- end }}
+
+{{/*
+Backups enabled
+*/}}
+{{- define "cluster.backupsEnabled" -}}
+{{- or (eq (include "cluster.cloudBackupsEnabled" .) "true") .Values.backups.volumeSnapshot.className -}}
```

Comment:
```suggestion
{{- or (eq (include "cluster.cloudBackupsEnabled" .) "true") (not (empty .Values.backups.volumeSnapshot.className )) -}}
```

Reply:
As far as I know string is false when empty, what this change improves?

Reply:
when I tested this locally and set my preferred classname it didn't evaluate to true as it should have.

Reply:
Will check thanks, and apply your suggestion

Reply:
Yeah, you right, it should be done like you proposed here.